aait/backend/group-2A/Domain/Entities/Comment.cs

```diff
@@ -8,8 +8,8 @@ public class Comment
         //The User That Commented
         public required int UserId { get; set; }
         public required int PostId { get; set; }
-        public required int CreatedAt{ get; set; }
-        public required int UpdatedAt{ get; set; }
+        public required DateTime CreatedAt{ get; set; }
+        public required DateTime UpdatedAt{ get; set; }
```

Comment:
Instead of having CreatedAt and UpdateAt fields in the Comment, you can create a BaseEntity with Id, createdAt and updatedAt fields . Then make all of the Entities tracked in the database as an entry to extend from the base entity.